key-vault-node-quickstart-v3/README.md

```diff
@@ -0,0 +1,192 @@
+
+Please select a Resource Group name and fill in the placeholder.
+The following example creates a resource group named *<YourResourceGroupName>* in the *eastus* location.
+
+```azurecli
+# To list locations: az account list-locations --output table
+az group create --name "<YourResourceGroupName>" --location "East US"
+```
+
+The resource group you just created is used throughout this tutorial.
+
+## Create an Azure Key Vault
+
+Next you create a Key Vault using the resource group created in the previous step. Although “ContosoKeyVault” is used as the name for the Key Vault throughout this article, you have to use a unique name. Provide the following information:
+
+* Vault name - **Select a Key Vault Name here**.
+* Resource group name - **Select a Resource Group Name here**.
+* The location - **East US**.
+
+```azurecli
+az keyvault create --name "<YourKeyVaultName>" --resource-group "<YourResourceGroupName>" --location "East US"
+```
+
+At this point, your Azure account is the only one authorized to perform any operations on this new vault.
+
+## Add a secret to key vault
+
+We're adding a secret to help illustrate how this works. You could be storing a SQL connection string or any other information that you need to keep securely but make available to your application. In this tutorial, the password will be called **AppSecret** and will store the value of **MySecret** in it.
+
+Type the commands below to create a secret in Key Vault called **AppSecret** that will store the value **MySecret**:
+
+```azurecli
+az keyvault secret set --vault-name "<YourKeyVaultName>" --name "AppSecret" --value "MySecret"
+```
+
+To view the value contained in the secret as plain text:
+
+```azurecli
+az keyvault secret show --name "AppSecret" --vault-name "<YourKeyVaultName>"
+```
+
+This command shows the secret information including the URI. After completing these steps, you should have a URI to a secret in an Azure Key Vault. Write this information down. You need it in a later step.
+
+## Clone the Repo
+
+Clone the repo in order to make a local copy for you to edit the source by running the following command:
+
+```
+git clone https://github.com/Azure-Samples/key-vault-node-quickstart.git
+```
+
+## Install dependencies
+
+Here we install the dependencies. Run the following commands:
+```
+cd key-vault-node-quickstart-v3
+```
+```
+npm install
+```
+
+This project used 2 node modules:
+
+* [ms-rest-azure](https://www.npmjs.com/package/ms-rest-azure) 
+* [azure-keyvault](https://www.npmjs.com/package/azure-keyvault)
+
+## Publish the web application to Azure
+
+Below are the few steps we need to do
+
+- The 1st step is to create a [Azure App Service](https://azure.microsoft.com/services/app-service/) Plan. You can store multiple web apps in this plan.
+
+    ```
+    az appservice plan create --name myAppServicePlan --resource-group myResourceGroup
```

Comment:
Casing for placeholders here is camel.  Like my comment below, please pick a casing and stick with it.